packages/jest-circus/src/formatNodeAssertErrors.ts

```diff
@@ -168,4 +168,12 @@ function assertionErrorMessage(
   );
 }
 
+function isAssertionError(error: Circus.TestError) {
+  return (
+    error instanceof AssertionError ||
+    (error &&
+      (error.name === AssertionError.name || error.code === 'ERR_ASSERTION'))
+  );
```

Comment:
could we do `return error && (error instanceof AssertionError || error.name === AssertionError.name || error.code === 'ERR_ASSERTION')`?

Reply:
changed it.